recipe/meta.yaml

```
@@ -11,13 +11,14 @@ source:
     - no_matlab.patch
 
 build:
-  number: 32
+  number: 33
   skip: true  # [win and vc<14]
```

Skip ppc64le, and try removing `win and vc<14` as mentioned at https://github.com/conda-forge/gmt-feedstock/pull/283#discussion_r1610882722

```suggestion
  skip: true  # [ppc64le]
```